OBAKitTests/Location/RegionsServiceTests.swift

```diff
@@ -83,26 +84,25 @@ class RegionsServiceTests: OBATestCase {
         stubRegions(dataLoader: dataLoader)
```

Comment:
Please update the function signature comment to `saved to disk` instead of `user defaults`.